src/ImageSharp/Processing/Processors/Quantization/EuclideanPixelMap{TPixel}.cs

```diff
@@ -149,13 +149,13 @@ private static int DistanceSquared(Rgba32 a, Rgba32 b)
     /// The granularity of the cache has been determined based upon the current
     /// suite of test images and provides the lowest possible memory usage while
     /// providing enough match accuracy.
-    /// Entry count is currently limited to 1185921 entries (2371842 bytes ~2.26MB).
+    /// Entry count is currently limited to 2335905 entries (4671810 bytes ~4.45MB).
     /// </para>
     /// </remarks>
     private unsafe struct ColorDistanceCache : IDisposable
     {
         private const int IndexBits = 5;
-        private const int IndexAlphaBits = 5;
+        private const int IndexAlphaBits = 6;
```

Comment:
This could be zero when `TPixel` doesn't have an alpha. It might be worth to eventually implement some trick to have two versions of EuclideanPixelMap, so we save memory when working with RGB pngs and gifs.

Reply:
Good point. We could easily preserve much more accurate matches in less space.